two-new-card-achievement-section

Darshna-Shrivastava opened the pull request.

Darshna-Shrivastava committed:
Relevant message

Darshna-Shrivastava commented on index.html:
```diff
@@ -506,6 +506,36 @@ <h3 class="name-body">
+                </h3>
+              </div>
+            </div>
+          </div>
```
New card has been added

ashish11chawda reviewed (changes requested):
Hey @Darshna-Shrivastava 
Thanks for updating the details
In my opinion it'll be great if you add these cards at the beginning of the container as these are latest updates

Sam-Ny reviewed (commented):
@Darshna-Shrivastava , Also it would be great if we can have one more card to add onto it, coz it is looking uneven . Ask @adivya-5 for more details.

Darshna-Shrivastava committed:
Added the cards at the beginning of achievementsec

The two new card are added at the end of the achievement section before now it has been added at the beginning of the achievement section because they are latest updates.

ashish11chawda reviewed (approved):
LGTM ✅
